apps/src/code-studio/pd/application_dashboard/quick_view.jsx

```diff
@@ -79,6 +72,7 @@ export default class QuickView extends React.Component {
         applications: data
       });
     });
+    this.setState({isWorkshopAdmin: this.props.route.isWorkshopAdmin});
```

Comment:
No need to add this to state since it doesn't change. Reference the prop directly.